plugins/terraformv2-backend/README.md

```diff
@@ -0,0 +1,14 @@
+# terraformv2
+
+Welcome to the terraformv2 backend plugin!
```

Comment:
I think we can just name it terraform-backend?  Applies to the name of this folder too and the package name too

Reply:
Renamed it.